airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -126,8 +127,18 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
                 f"The following streams are partially available: {[s.name for s in partially_available_streams]}, "
                 f"add the following scopes to download all available data: {required_scoped}"
             )
+
+            if "crm.schemas.custom.read" in granted_scopes and "crm.objects.custom.read" in granted_scopes:
+                available_streams.extend(self.get_custom_object_streams(api=api, common_params=common_params))
```

Comment:
It seems that your feature is not supported if an API Key used for authentication. Is it intentional?

Reply:
I think Hubspot is deprecating API Keys, so I don't know if this can be supported with that. My personal hubspot account can't create any new API Key to test that, so I don't know if I should allow that.

At my company, we have been running a custom version of this connector for a while and it has worked with OAuth tokens.